su-di-noi/valori.md

```diff
@@ -1,74 +1,72 @@
-# I valori che portiamo avanti
+# I nostri valori
 
-Ognuno di noi sarà chiamato nello svolgimento del proprio lavoro ad avere un rapporto con la 
-clientela, siano essi proprietari del business, project manager o sviluppatori/designer/marketer. Il 
-rapporto che si instaura con il cliente è fondamentale. Nel momento della produzione del servizio di 
-consulenza bisogna essere in grado di tenere presenti tutti i valori e gli standard che Nebulab si 
-propone, rispettivamente, di rappresentare e di mantenere. Lavorare per i clienti vuol dire spesso 
-anche lavorare insieme ad uno o più membri del team di Nebulab. È importante supportarsi e 
-coordinarsi, sopperendo ad eventuali mancanze degli altri membri del team.
+Tutti noi, nello svolgimento del nostro lavoro, abbiamo rapporti diretti con i nostri clienti. Il
+rapporto che si instaura con il cliente è fondamentale: dobbiamo sempre tenere presenti tutti i
+valori e gli standard che Nebulab si propone di rappresentare e di mantenere.
 
-Tutto ciò che Nebulab si propone di essere è riportato sul nostro sito internet. Un cliente che ci 
-contatta porta con sé un bagaglio di aspettative dovute alla comunicazione presente sul sito web che 
-incorpora dei concetti chiave come: 
+Lavorare per i clienti vuol dire spesso anche lavorare insieme a uno o più membri del team di 
+Nebulab. È importante supportarsi e coordinarsi, sopperendo a eventuali mancanze degli altri membri 
+del team.
 
-- persone appassionate di tecnologia e del web
-- giovani ed esperti
-- sviluppiamo prodotti di qualità
-- risolviamo problemi
-- nerd comunicativi
-- persone che amano lavorare e divertirsi
-- attitudine alla sperimentazione e alla condivisione
+Tutto ciò che ci proponiamo di essere è riportato sul nostro sito internet, dunque un cliente che ci 
```

Comment:
cambierei con: Tutto ciò che ci proponiamo di essere è riportato sul nostro sito internet. Chi entra in contatto con noi si aspetta di...